src/libs/browser-storage.js

```diff
@@ -161,6 +161,7 @@ export const staticStorageSlot = (storage, key) => {
     setStatic(storage, key, newValue);
     next(newValue);
   };
+
   listenStatic(storage, key, next);
-  return { subscribe, get, set, update: (fn) => set(fn(get())) };
+  return { subscribe, get, set, update: (fn) => set(fn(get())), reset: () => set(storage) };
```

Comment:
This looks incorrect. It looks like this is trying to set a value in a Storage object to the Storage object itself.

I would suggest capturing the initial value when the "slot" is instantiated and using that to reset it.

```suggestion
  const initialValue = get();
  return { subscribe, get, set, update: (fn) => set(fn(get())), reset: () => set(initialValue) };
```